src/sentry/api/serializers/models/event.py

```diff
@@ -265,5 +266,6 @@ def serialize(self, obj, attrs, user):
                 'id': obj.user_id,
                 'email': obj.email,
                 'username': obj.username,
+                'ipAddress': obj.ip_address,
```

Comment:
Can we use `ip_address` so it's consistent with our current user model?

Reply:
hmm which user model? it looks like the event user serializer does the same thing: https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/eventuser.py#L21

Reply:
Ahh, I only searched for frontend usage and it looks like we use a mix of both: 

![image](https://user-images.githubusercontent.com/79684/48383102-419d1b00-e699-11e8-80ec-050a54d92c59.png)

![image](https://user-images.githubusercontent.com/79684/48383115-4d88dd00-e699-11e8-8064-9044dc1cdfe6.png)

Reply:
oh weird. it looks like in all of our serializers (EventUser, UserIPSerializer, AuditLogEntrySerializer) we camel case ipAddress. It's kind of hard to tell if there's another place that'd be returning `ip_address` since searching for that beyond just the serializers directory yields too many results to reasonably check.

up to you -- i'm happy to make it either.

Reply:
Let's keep it consistent with the serializers then; I'll try to track down where we use the underscored name.